### ISSUE TITLE
Zendesk ticket with 'On-hold' system status fails import

### ISSUE THREAD

**Issue by anuzhin** [zammad/zammad#2439]
### Infos:

* Used Zammad version: 2.8
* Installation method (source, package, ..): any
* Operating system: any
* Database + version: any
* Elasticsearch version: any
* Browser + version: any

### Expected behavior:

* Migration from Zendesk competed without errors

### Actual behavior:

* Error when migrating tickets. Logs:
```
E, [2019-01-17T02:46:51.179615 #2279-47349039994100] ERROR -- : ImportJob 'Import::Zendesk' failed: undefined method `id' for nil:NilClass

E, [2019-01-17T02:46:51.179701 #2279-47349039994100] ERROR -- : undefined method `id' for nil:NilClass (NoMethodError)

/opt/zammad/lib/sequencer/unit/import/zendesk/ticket/state_id.rb:13:in `state_id'

/opt/zammad/lib/sequencer/unit/common/provider/attribute.rb:27:in `value'

/opt/zammad/lib/sequencer/unit/common/provider/attribute.rb:32:in `ignore?'

/opt/zammad/lib/sequencer/unit/common/provider/attribute.rb:8:in `process'

/opt/zammad/lib/sequencer/unit/base.rb:202:in `process'

/opt/zammad/lib/sequencer.rb:77:in `block (4 levels) in process'

/opt/zammad/lib/mixin/start_finish_logger.rb:7:in `log_start_finish'
```

### Steps to reproduce the behavior:

* Enable On-hold ticket status in Zendesk
* Create ticket with On-hold status in Zendesk
* Run migration

### Additional Information:

About On-hold ticket status: https://support.zendesk.com/hc/en-us/articles/203661576
Name of status in API JSON: 'hold'

**Comment by thorsteneckel** [comment]
Hi @anuzhin - thanks for reaching out and providing the detailed information. Since we don't have an equivalent of "On Hold" in Zammad I was wondering how to map it 🤔 We have two options:

- Import as "Open"
- Import as "Pending reminder" - but with which date?

What do you think?

**Comment by anuzhin** [comment]
@thorsteneckel , I think what "On Hold" status in our case can be replaced on "Open" and new tag

**Comment by thorsteneckel** [comment]
Hi @anuzhin - Thanks for providing the fix and kudos for figuring things out 💪 This will be available in `develop` in about 20 minutes from now and released to stable with Zammad 2.9 🚀 Closing now.
